src/startupscriptgenerator/dotnetcore/main.go

```diff
@@ -14,14 +14,16 @@ import (
 func main() {
 	common.PrintVersionInfo()
 
+	appPathPtr := flag.String(
```

Comment:
Following a similar pattern as other platforms' startup script generators. This simplifies the scenario for dotnetcore too without the need for figuring out the output directory.